game/world/objects/npc.cpp

```diff
@@ -3091,9 +3113,19 @@ void Npc::setToFistMode() {
   }
 
 void Npc::aiPush(AiQueue::AiAction&& a) {
-  if(a.act==AI_OutputSvmOverlay)
-    aiQueueOverlay.pushBack(std::move(a)); else
-    aiQueue.pushBack(std::move(a));
+  switch(a.act) {
+    case AI_OutputSvmOverlay:
+      aiQueueOverlay.pushBack(std::move(a));
+      break;
+    case AI_QuickLook:
```

Comment:
This looks like a partial workaround for:
https://github.com/Try/OpenGothic/issues/523

My understanding, that some commands, such as `AI_QuickLook` has to run in dedicated queue, with own command-level tracking